libcnb-cargo/src/package/command.rs

```diff
@@ -25,21 +25,30 @@ pub(crate) fn execute(args: &PackageArgs) -> Result<()> {
 
     let current_dir = std::env::current_dir().map_err(Error::GetCurrentDir)?;
 
-    let workspace = get_cargo_workspace_root(&current_dir)?;
+    let workspace_root_path = get_cargo_workspace_root(&current_dir)?;
 
-    let workspace_target_dir = MetadataCommand::new()
-        .manifest_path(&workspace.join("Cargo.toml"))
+    let cargo_metadata = MetadataCommand::new()
+        .manifest_path(&workspace_root_path.join("Cargo.toml"))
         .exec()
-        .map(|metadata| metadata.target_directory.into_std_path_buf())
         .map_err(|e| Error::ReadCargoMetadata {
-            path: workspace.clone(),
+            path: workspace_root_path.clone(),
             source: e,
         })?;
 
+    let package_dir = args.package_dir.clone().unwrap_or_else(|| {
+        cargo_metadata
+            .workspace_root
+            .into_std_path_buf()
+            .join("libcnb-packaged")
```

Comment:
Ah we probably need to update the example `cargo libcnb package` log output here at some point:
https://github.com/heroku/libcnb.rs/blob/e3a3da8fecfd08b4815a782ba2dc449d71e8f6e9/README.md?plain=1#L188-L190
https://github.com/heroku/libcnb.rs/blob/e3a3da8fecfd08b4815a782ba2dc449d71e8f6e9/libcnb-cargo/README.md?plain=1#L26-L28

Reply:
We should do that in this PR I think. I'm on it!